homeassistant/components/palazzetti/__init__.py

```diff
@@ -0,0 +1,40 @@
+"""The Palazzetti integration."""
+
+from __future__ import annotations
+
+from pypalazzetti.exceptions import CommunicationError, ValidationError
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryError
+
+from .const import DOMAIN
+from .coordinator import PalazzettiDataUpdateCoordinator
+
+PLATFORMS: list[Platform] = [Platform.CLIMATE]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
```

Comment:
Please extend the ConfigEntry type with what you are putting into `entry.runtime_data`